### PR TITLE
Added "Map to Target" directive to column actions dropdown.

### PR DESCRIPTION
**Pull request by mukhlin**
This is a first step to add "Map to Target" to columns action dropdown. Next step is to implement data model, table and field selection.

### CODE REVIEW

File: cdap-ui/app/cdap/services/ThemeHelper.ts [hunk 1]
```diff
@@ -344,7 +344,6 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
           'wrangler-datamodel-viewer'
         );
       }
-
```
Comment (Biao) [inline cdap-ui/app/cdap/services/ThemeHelper.ts:347]: let's avoid unnecessary changes to the CDAP code unless it is recommended by formatting tool.

File: cdap-ui/app/cdap/services/ThemeHelper.ts [hunk 2]
```diff
@@ -379,6 +378,7 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
       showDashboard: true,
       showReports: true,
       showDataPrep: true,
+      // showWranglerDatamodelViewer: true,
```
Comment (Biao) [inline cdap-ui/app/cdap/services/ThemeHelper.ts]: Assuming this is for testing in local development env, it would be great to avoid landing these code into upstream code base. I would remove it in this PR and only keep it in my local branch.

File: cdap-ui/app/cdap/services/ThemeHelper.ts [hunk 3]
```diff
@@ -419,7 +419,6 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
     ) {
       features.showWranglerDatamodelViewer = featuresJson['wrangler-datamodel-viewer'];
     }
-
```
Comment (Biao) [inline cdap-ui/app/cdap/services/ThemeHelper.ts:422]: ditto. let's avoid unnecessary changes.

File: cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js [hunk 4]
```diff
@@ -78,15 +80,30 @@ export default class ColumnActionsDropdown extends Component {
...
         tag: ParseDirective,
         requiredColCount: 1,
       },
       {
         tag: 'divider',
-      },
+      }
+    );
+    if (Theme.showWranglerDatamodelViewer) {
+      this.directives.push(
```
Comment (Biao) [inline cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js:95]: We should create a flag called wrangler-data-model and use that flag to guard this feature.  All of our code should be hide behind that flag. wrangler-datamodel-viewer should be removed once we are done.

File: cdap-ui/app/cdap/services/ThemeHelper.ts (hunk 1 above)
Comment (mukhlin) [reply cdap-ui/app/cdap/services/ThemeHelper.ts:347]: Ok, I will take it into account in the future.

File: cdap-ui/app/cdap/services/ThemeHelper.ts (hunk 3 above)
Comment (mukhlin) [reply cdap-ui/app/cdap/services/ThemeHelper.ts:422]: Ok, I will take it into account in the future.

File: cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js (hunk 4 above)
Comment (mukhlin) [reply cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js:95]: I didn't get what you mean. This flag has been created already, I just used it.

File: cdap-ui/app/cdap/services/ThemeHelper.ts (hunk 2 above)
Comment (mukhlin) [reply cdap-ui/app/cdap/services/ThemeHelper.ts]: Ok, I will remove it.

Comment (mukhlin) [reply cdap-ui/app/cdap/services/ThemeHelper.ts]: Done.

File: cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js (hunk 4 above)
Comment (mukhlin) [reply cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js:95]: Added new `wrangler-data-model` flag.

**Review by Biao — APPROVED** [review]
looks good to me. I am going to request someone from CDAP team to take a look before merge.

We will want to start to investigate how to add integration test soon once we started to add more logic.
